cli/UploadSourcemaps.ts

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
+import { Command, Option } from 'commander';
+import FormData from 'form-data';
+import fs from 'fs';
+import path from 'path';
+
+export const uploadSourcemapsCommand: Command = new Command();
+
+uploadSourcemapsCommand
+  .name('upload-sourcemaps')
+  .addOption(
+    new Option('-p, --platform <value>', 'Platform')
+      .choices(['ios', 'android'])
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option(
+      '-d, --dir <value>',
+      'The path of the directory including the source map file',
+    ).makeOptionMandatory(),
+  )
+  .addOption(
+    new Option('-t, --token <value>', 'Your App Token')
+      .env('INSTABUG_APP_TOKEN')
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option('-v, --name <value>', 'The app version name')
+      .env('INSTABUG_APP_VERSION_NAME')
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option('-c, --code <value>', 'The app version code')
+      .env('INSTABUG_APP_VERSION_CODE')
+      .makeOptionMandatory(),
+  )
+  .addOption(new Option('-l, --label <value>', "The CodePush label if it's a CodePush release"))
+  .action(function (this: Command) {
+    const options = this.opts<UploadSourcemapsOptions>();
+    uploadFile(options);
+  })
+  .showHelpAfterError();
+
+interface UploadSourcemapsOptions {
```

Comment:
**suggestion:** Let move the interface definition to the top of the file, right after the imports, since it's used in previous lines of code.